tests/python/frontend/onnx/test_forward.py

```diff
@@ -142,6 +142,37 @@ def test_reshape():
     tvm.testing.assert_allclose(ref_shape, tvm_out.shape)
 
 
+def test_expand():
```

Comment:
You will also need to call this function at the bottom of the file

Reply:
ok, I'll move it.

Reply:
@soiferj I added the call.